docs/index.asciidoc

```diff
@@ -76,6 +89,10 @@ A client that receives an HTTP request timeout is expected to retry the entire r
 When this plugin is blocked, it will reject _new_ requests with HTTP `429 Too Many Requests`.
 Clients that receive `429`-s are expected to wait a moment before retrying the request — exponential back-off is encouraged to ease flood recovery.
 
+//
+// END: Elastic-internal implementation details
+////
```

Comment:
Formatting works as expected.  (Yeah, I know that's what we expected, but that's always what we get.). :-)